src/hooks/useFiatText.js

```diff
@@ -49,7 +51,7 @@ export const useFiatText = (props: Props): string => {
         })
       : '0'
 
-  const fiatSymbol = `${getSymbolFromCurrency(isoFiatCurrencyCode)}${fiatSymbolSpace ? ' ' : ''}`
+  const fiatSymbol = hideFiatSymbol ? '' : `${getSymbolFromCurrency(isoFiatCurrencyCode)}${fiatSymbolSpace ? ' ' : ''}`
```

Comment:
'hideFiatSymbol' can be either a boolean of an undefined. 

So either set the default value of 'hideFiatSymbol' to false or explicitly test for `hideFiatSymbol === true'.
This is something typescript would give an error about.